bsc#1062247 add new svg logo

MaximilianMeister opened the pull request:
with the full name SUSE® CaaS Platform

bsc#1062247

Signed-off-by: Maximilian Meister <mmeister@suse.de>

also the login page and the normal layout header will use the svg

~~there is only still one issue that the navbar in the app and in the login don't share the same css it seems, i have tweaked the image size in the application css but maybe that;'s not the best way to do it. @vitoravelino do you have an idea?~~

updated to `height: 60px` as per @vitoravelino's suggestion

![logologin](https://user-images.githubusercontent.com/5364817/31379322-2ea9a9fc-adae-11e7-831f-1b5da7cbb68c.png)
![logoapp](https://user-images.githubusercontent.com/5364817/31379329-32158174-adae-11e7-88db-27f19f913eb2.png)

vitoravelino commented:
@MaximilianMeister It doesn't match the previous font family. I'm not sure if this is an issue or not. If not, great! For dex, what I did was convert the svg to png. Remember to set the height as 60px and it will work out of the box when we replace the existent one.

Thank you for this!

MaximilianMeister commented:
> It doesn't match the previous font family. I'm not sure if this is an issue or not. If not, great!

i think it's ok as it it unified now, before we had one unknown font in the login screen as svg, and another standard font as text in the app. but suggestions for another font are also welcome

> For dex, what I did was convert the svg to png. 

https://github.com/kubic-project/dex-branding/pull/2

> Remember to set the height as 60px and it will work out of the box when we replace the existent one.

thanks for the hint, i dropped the width and added the 60px height and it seems to work

vitoravelino commented:
> i think it's ok as it it unified now, before we had one unknown font in the login screen as svg, and another standard font as text in the app. but suggestions for another font are also welcome

I think it's okay to merge without matching the previous font-family but it would be nice to check with the branding team just to be sure. I'm not sure how strict SUSE is with this kind of stuff. We can always update it later.

I'll pull the changes locally and review now.

vitoravelino commented:
Weird... In your screenshots the font is completely different from the one rendered here.

vitoravelino commented:
> Weird... In your screenshots the font is completely different from the one rendered here.

I think I don't have Roboto installed in my computed and it renders with sans-serif. In the previous one there was no font-family thing. I think it's related to how the svg is exported. I don't understand these things, so can't help.

MaximilianMeister commented:
> I think I don't have Roboto installed in my computed and it renders with sans-serif. In the previous one there was no font-family thing. I think it's related to how the svg is exported. I don't understand these things, so can't help.

i dont know either, i thought svg is just paths and has no further dependencies...
